docs/machine-learning/tutorials/mlnet-cli.md

```diff
@@ -200,7 +200,7 @@ Those enumerated assets are explained in the following steps of the tutorial.
 
     Run the console app from the command-prompt by typing the following commands:
 
-     ```
+     ```console
```

Comment:
Not sure if `bash` is more appropriate than `console` here.

Reply:
I made this change because I think you're definitely right. I didn't spot that it was a mac example that specified bash until I looked at the current page. Thanks!